src/DoubleLinkedList.sol

```diff
@@ -99,47 +96,19 @@ library DoubleLinkedList {
         if (id == address(0)) revert AddressIsZero();
         if (list.accounts[id].value != 0) revert AccountAlreadyInserted();
 
-        uint256 numberOfIterations;
-        address next = list.head; // If not added at the end of the list `id` will be inserted before `next`.
+        address next = list.accounts[address(0)].next; // `id` will be inserted before `next`.
```

Comment:
```suggestion
        address next = getHead(list); // `id` will be inserted before `next`.
```

Reply:
But maybe it adds another JUMP and it complexifies the bytecode only for readability